internal/app/cf-terraforming/cmd/root.go

```diff
@@ -130,6 +134,14 @@ func initConfig() {
 }
 
 func persistentPreRun(cmd *cobra.Command, args []string) {
+
+	accountID = viper.GetString("account")
+	if accountID == "" {
+		// backward-compatible with organization
+		accountID = viper.GetString("organization")
+	}
+
+	if apiToken = viper.GetString("token"); apiToken == "" {
```

Comment:
I think we're missing the log error / closing if bracket here?

Reply:
Nm: diff view weirdness.

Reply:
It was because the diff view cannot show the indentations nicely.